### PR TITLE
UI/seller/guide page

### PR DESCRIPTION
**Pull request by AYAszk**
[UI/seller/guidepage] - Added guidepage.blade.php

### CODE REVIEW

File: composer.json
Comment (Ditzler01) [inline composer.json:1]: Did you install a bootstrap library?
Did you run a command like npm install?

**Review by Ditzler01 — CHANGES_REQUESTED** [review]
Please review my comments. You can reply to each comment here in GitHub.
If you have questions regarding to my comments, please reach out to me in slack.

File: routes/web.php [hunk 1]
```diff
@@ -20,3 +22,6 @@
 Auth::routes();
 
 Route::get('/home', [App\Http\Controllers\HomeController::class, 'index'])->name('home');
+Route::get('/sellers/guide', [SellerController::class, 'guide'])->name('guide');
+Route::get('/buyers/service', [BuyerController::class, 'service'])->name('service');
+Route::get('/buyers/help', [BuyerController::class, 'help'])->name('help');
```
[the remark this answers is not in the document]
Comment (AYAszk) [reply routes/web.php:27]: I have a question. Are you suggesting that each time I create a new branch, I should commit with only the route name written in web.php?

File: composer.json
Comment (AYAszk) [reply composer.json:1]: Yes, I have already done it.

File: routes/web.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (AYAszk) [reply routes/web.php]: I may have resolved it. It seems conflicts occurred, so it's okay to have just one route name, isn't it?

**Review by Ditzler01 — APPROVED** [review]

**Pull request merged**